config/routes.rb

```diff
@@ -63,14 +63,15 @@
   get 'api/savePolyLineData', to: 'api/api#savePolyLineData'
   post 'api/savePolyLineData', to: 'api/api#savePolyLineData'
   post '/api/busStopData', to: 'api/api#busStopData'
+  post 'api/geoCodeBusstops', :to => 'api/api#geoCodeBusstops'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.